### PR TITLE
Refactor code to the context of trackbeau project

### PR DESCRIPTION
**Pull request by hanhuiice**
Rename classes, variables and package names to match the use case of the project.

### CODE REVIEW

File: src/test/java/seedu/trackbeau/logic/parser/EditCommandParserTest.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + INVALID_NAME_DESC, Name.MESSAGE_CONSTRAINTS); // invalid name
         assertParseFailure(parser, "1" + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS); // invalid phone
         assertParseFailure(parser, "1" + INVALID_EMAIL_DESC, Email.MESSAGE_CONSTRAINTS); // invalid email
-        assertParseFailure(parser, "1" + INVALID_ADDRESS_DESC, Address.MESSAGE_CONSTRAINTS); // invalid address
+        assertParseFailure(parser, "1" + INVALID_ADDRESS_DESC, Address.MESSAGE_CONSTRAINTS); // invalid trackbeau
```
Comment (isabelteo) [inline src/test/java/seedu/trackbeau/logic/parser/EditCommandParserTest.java:87]: I think its supposed to be address, same for the comment in line 155

File: src/test/java/seedu/trackbeau/logic/commands/EditCommandTest.java [hunk 2]
```diff
@@ -36,47 +36,47 @@
...
+        Customer editedCustomer = new PersonBuilder().build();
+        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedCustomer).build();
         EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
 
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedPerson);
+        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedCustomer);
 
-        Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
-        expectedModel.setPerson(model.getFilteredPersonList().get(0), editedPerson);
+        Model expectedModel = new ModelManager(new TrackBeau(model.getAddressBook()), new UserPrefs());
```
Comment (isabelteo) [inline src/test/java/seedu/trackbeau/logic/commands/EditCommandTest.java:45]: getAddressBook() method

File: src/test/java/seedu/trackbeau/logic/commands/EditCommandTest.java [hunk 3]
```diff
@@ -85,23 +85,23 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
     public void execute_filteredList_success() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
-        Person personInFilteredList = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        Person editedPerson = new PersonBuilder(personInFilteredList).withName(VALID_NAME_BOB).build();
+        Customer customerInFilteredList = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
```
Comment (isabelteo) [inline src/test/java/seedu/trackbeau/logic/commands/EditCommandTest.java:88]: getFilteredPersonList()

File: src/main/java/seedu/trackbeau/storage/JsonTrackBeauStorage.java [hunk 4]
```diff
@@ -60,16 +60,16 @@ public Path getAddressBookFilePath() {
     }
 
     @Override
-    public void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException {
+    public void saveAddressBook(ReadOnlyTrackBeau addressBook) throws IOException {
```
Comment (isabelteo) [inline src/main/java/seedu/trackbeau/storage/JsonTrackBeauStorage.java:63]: The method and parameter names contain AddressBook

File: docs/tutorials/TracingCode.md [hunk 5]
```diff
@@ -189,22 +189,22 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
    @Override
    public CommandResult execute(Model model) throws CommandException {
        ...
-       Person personToEdit = lastShownList.get(index.getZeroBased());
-       Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
-       if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
+       Person customerToEdit = lastShownList.get(index.getZeroBased());
+       Person editedCustomer = createEditedPerson(customerToEdit, editPersonDescriptor);
+       if (!customerToEdit.isSamePerson(editedCustomer) && model.hasPerson(editedCustomer)) {
```
Comment (isabelteo) [inline docs/tutorials/TracingCode.md:194]: hasPerson() method

File: docs/tutorials/TracingCode.md [hunk 6]
```diff
@@ -189,22 +189,22 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
...
            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
        }
-       model.setPerson(personToEdit, editedPerson);
+       model.setPerson(customerToEdit, editedCustomer);
```
Comment (isabelteo) [inline docs/tutorials/TracingCode.md:197]: setPerson() method

File: src/main/java/seedu/trackbeau/model/customer/Customer.java [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
-import seedu.address.model.tag.Tag;
+import seedu.trackbeau.model.tag.Tag;
 
 /**
- * Represents a Person in the address book.
+ * Represents a Person in the trackbeau book.
```
Comment (flairekq) [inline src/main/java/seedu/trackbeau/model/customer/Customer.java:13]: I think need to change Person to Customer here too.

**Review by flairekq — COMMENTED** [review]
Overall LGTM just a few cosmetic changes required.

**Pull request merged**